nautilus_trader/adapters/interactive_brokers/client/market_data.py

```diff
@@ -874,3 +874,52 @@ async def process_historical_ticks(self, *, req_id: int, ticks: list, done: bool
         if not done:
             return
         await self._process_trade_ticks(req_id, ticks)
+
+    async def get_price(self, contract, tick_type="MidPoint"):
+        """
+        Helper function to request market data for a specific contract and tick type.
+        """
+        # Generate Request ID
+        req_id = self._next_req_id()
+        # Create a Request object and add it to the _requests
+        request = self._requests.add(
+            req_id=req_id,
+            name=f"{contract.symbol}-{tick_type}",
+            handle=functools.partial(self._eclient.reqMktData, req_id, contract, tick_type, False, False, []),
+            cancel=functools.partial(self._eclient.cancelMktData, req_id)
+        )
+        # Request Market Data
+        request.handle()
+        # Await Request
+        result = await self._await_request(request, timeout=60)
+        return result
```

Comment:
```suggestion
        return await self._await_request(request, timeout=60)
```

Nit: we don't need to define `result` since we are just returning the value and not accessing that variable elsewhere in the method